buildenv/jenkins/JenkinsfileBase

```diff
@@ -126,6 +126,7 @@ def setupParallelEnv() {
 				//ToDo: need to find a better way to pass parameters to downstream builds.
 				//Ideally, we should only need to pass in modified parameters (i.e.,
 				//BUILD_LIST, IS_PARALLEL)
+				JENKINS_FILE = params. JenkinsFile ? params.JenkinsFile : ""
```

Comment:
- Please remove the space in `params. JenkinsFile`
- Please put this line in `setupEnv()`
- Please use the grinder to verify the change and provide the links

Reply:
The changes to the code were made as per your recommendations.
The test link for the launcher is the following: https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder_Sandbox3/951/
The test link for the first subtest is the following: https://hyc-runtimes-jenkins.swg-devops.com/view/Test_grinder/job/Grinder_Sandbox3/952/
As you can see in the parameters, the JenkinsFile field is not the default, and is the same for both.